controllers/users.js

```diff
@@ -69,48 +81,27 @@ module.exports.patchMe = async (req, res, next) => {
   } catch (e) {
     next(e);
   }
-};
+});
 
-// проверка пользователя
-// module.exports.login = (req, res) => {
-//   const { email, password } = req.body;
-//   return User.findUserByCredentials(email, password)
-//     .then((user) => {
-//       const token = jwt.sign(
-//         { _id: user._id },
-//         'some-secret-key',
-//         { expiresIn: '7d' },
-//       );
-//       res.cookie('jwt', token, {
-//         maxAge: 3600000 * 24 * 7,
-//         httpOnly: true,
-//         sameSite: true,
-//       });
-//       res.status(200).send(token);
-//     })
-//     .catch((err) => {
-//       res
-//         .status(401)
-//         .send({ message: err.stack });
-//     });
-// };
-
-module.exports.login = async (req, res, next) => {
+// Логировние
+module.exports.login = (async (req, res, next) => {
   try {
     const { email, password } = req.body;
     const user = await User.findUserByCredentials(email, password);
-    const token = jwt.sign(
+    const token = await jwt.sign(
       { _id: user._id },
-      'some-secret-key',
+      JWT_SECRET,
       { expiresIn: '7d' },
     );
-    res.cookie('jwt', token, {
-      maxAge: 3600000 * 24 * 7,
-      httpOnly: true,
-      sameSite: true,
-    });
-    res.status(200).send(token);
+    res.cookie('jwt', token, JWT_SECRET)
+      .cookie({
+        maxAge: 3600000 * 24 * 7,
+        httpOnly: true,
+        secure: true,
+        sameSite: true,
+      });
```

Comment:
Можно лучше: поправить постановку куки.
Я уже писал, что `res.cookie()` ожидает на вход три параметра: `name`, `value`,`options`. Похоже, что здесь случайная ошибка. 
К тому же использование `secure` означает, что кука будет установлена только если запрос был отправлен по протоколу `HTTPS`.

В связи с этим рекомендую вынести настройки для установки куки в `appconfig.js`. А здесь оставить так:
```suggestion
    res.cookie('jwt', token, JWT_COOKIE_OPTIONS);
```

Общие настройки будут выглядеть как-то так:
```js
const JWT_COOKIE_OPTIONS = {
  maxAge: 3600000 * 24 * 7,
  httpOnly: true,
  secure: NODE_ENV === 'production',
  sameSite: true,
};
```

Обратите внимание, что `secure` будет включен только для продакшена, те для сервера, который выложен.